src/platform/packages/private/kbn-reporting/common/routes.ts

```diff
@@ -24,6 +24,10 @@ export const INTERNAL_ROUTES = {
     DELETE_PREFIX: prefixInternalPath + '/jobs/delete', // docId is added to the final path
     DOWNLOAD_PREFIX: prefixInternalPath + '/jobs/download', // docId is added to the final path
   },
+  SCHEDULED: {
+    LIST: prefixInternalPath + '/scheduled/list',
```

Comment:
Just out of curiosity: why not `GET [...]/schedules`? (à la RESTful)

Reply:
I was following the existing list jobs route that is `/jobs/list` but I'm open to changing if you feel strongly about it :)